public/index.html

```diff
@@ -2,14 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/map-marked-alt-solid.svg"/>
+    <script src="https://kit.fontawesome.com/419f457776.js" crossorigin="anonymous"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" content="#5B1B52" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="App To Show You Wikipedia Entry Of What is Nearby"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="/map-marked-alt-solid.png" />
```

Comment:
I don't think we need a forward slash here, and the path to the icon should just be  ``<link rel="apple-touch-icon" href="map-marked-alt-solid.png" />``

Reply:
Thank you for saying something! Change has been made.